test/CMakeLists.txt

```diff
@@ -42,7 +42,8 @@ FetchContent_MakeAvailable(googletest)
 include(GoogleTest)
 
 set(ROCWMMA_TEST_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR})
-set(ROCWMMA_COMMON_TEST_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/hip_device.cpp)
+set(ROCWMMA_COMMON_TEST_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/hip_device.cpp
+                                ${CMAKE_CURRENT_SOURCE_DIR}/rocwmma_gtest_main.cpp)
```

Comment:
Since we are now creating our own gtest main, don't forget to remove the linkage to gtest_main library.